doc: add minutes for meeting 16 July 2025 (#1767)
nodejs/TSC: meetings/2025-07-16.md

# Node.js Technical Steering Committee (TSC) Meeting 2025-07-16

## Links

* **Recording**:  <https://youtube.com/live/FOQLs8_HWrs>
* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1766>

## Present

* Antoine du Hamel @aduh95 (voting member)
* Gireesh Punathil @gireeshpunathil (voting member)
* James Snell @jasnell (voting member)
* Chengzhong Wu @legendecas (voting member)
* Marco Ippolito @marco-ippolito (voting member)
* Matteo Collina @mcollina (voting member)
* Michael Dawson @mhdawson (voting member)
* Filip Skokan @panva (voting member)
* Rafael Gonzaga @RafaelGSS (voting member)
* Darshan Sen @RaisinTen (voting member)
* Richard Lau @richardlau (voting member)
* Ruy Adorno @ruyadorno (voting member)
* Michaël Zasso @targos (voting member)
* Robin - Guest OpenJS Executive director

> * Claudio - Guest - website team member and more

Reviewer: ```suggestion
* Claudio Wunder @ovflowd (Website team member)
```

  * Claudio outlined the proposal to highlight our partners/supporters
  * Would co-ordinate with the Foundation marketing team
  * Robin, as part of the sovereign tech fund, we formalized agreements with
    supporting companies. When we bring on a new partner
  * Michael - please cross check against <https://github.com/nodejs/build>
  * Michael, any concerns from the TSC members who are here.

* Dedicated EOL page [#7906](https://github.com/nodejs/nodejs.org/issues/7906)
  * Claudio - good to have a dedicated static page, and then links to ESP partners
  * Michael - I think we agreed to work towards to this as a step after changing the link on the
    main page to Matteo’s blog post. Vue has a similar page.
  * Claudio - at this point just asking to see if there are any concerns
  * Matteo - something that might be interesting to include are the vulnerabilities which are not
    patch in the EOL versions.
  * Filip - like idea, however, not certain about displaying all of the details, but maybe just list the
    CVEs and severity. Could possibly have a link.
  * Claudio would need to have list

>   * Filip can probably use same dataset that isMyNode vulnerable uses

Reviewer 2: ```suggestion
  * Filip - it can use same dataset that [is-my-node-vulnerable](https://github.com/nodejs/is-my-node-vulnerable) uses
```
